src/Codeception/Module/MongoDb.php

```diff
@@ -148,13 +204,46 @@ protected function loadDump()
         if ($this->isDumpFileEmpty) {
             return;
         }
+        switch ($this->config['dump_type']) {
```

Comment:
I think you can wrap this into try-catch and not use `loadDump*` methods here. As they are one liner + catching exceptions